piq/functional.py

```diff
@@ -0,0 +1,72 @@
+from typing import Tuple
+
+import torch
+
+
+def ifftshift(x: torch.Tensor):
+    """ Similar to np.fft.ifftshift but applies to PyTorch Tensors
+    """
+    shift = [-(ax // 2) for ax in x.size()]
+    return torch.roll(x, shift, tuple(range(len(shift))))
+
+
+def get_meshgrid(size: Tuple[int, int]) -> torch.Tensor:
+    """
```

Comment:
We use raw docstring everywhere 
```suggestion
    r"""
```